include/seqan3/alignment/pairwise/detail/policy_affine_gap_recursion.hpp

```diff
@@ -220,7 +222,14 @@ class policy_affine_gap_recursion
      */
     score_type lowest_viable_score() const noexcept
     {
-        assert(gap_open_score <= 0 && gap_extension_score <= 0);
+        if constexpr (simd_concept<score_type>)
+            assert(gap_open_score[0] <= 0 && gap_extension_score[0] <= 0);
+        else
+            assert(gap_open_score <= 0 && gap_extension_score <= 0);
+
+        return maybe_convert_to_simd(std::numeric_limits<original_score_type>::lowest()) -
+               (gap_open_score + gap_extension_score);
+    }
```

Comment:
No this is ok in this commit, because in the first we are not calling the interface yet. So, this is the first commit that makes the change necessary.